config/locales/en.yml

```diff
@@ -682,6 +682,9 @@ en:
       hero:
         desc_html: Displayed on the frontpage. At least 600x100px recommended. When not set, falls back to server thumbnail
         title: Hero image
+      invite_request_hint:
+        desc_html: A hint shows above the reason for join in textarea. You can use HTML tags
```

Comment:
I don’t think it’s a good idea anymore to let admins use HTML tags directly. This makes the strings unusable for the API and apps.

Reply:
I considered the condition that the admins may need to write some external links and need something like `<strong>`, `<br>` for better formating and highlighting. 

As you said, it's truth that HTML has poor support on app, maybe can we just consider limiting HTML to several HTML tags like we to on status content formatter?
And also, the same rules should be applied to `site_extended_description`?